lib/helpers/pptx-content-helper.js

```diff
@@ -35,6 +35,30 @@ class PptxContentHelper {
 
         return objectInfo;
     }
+
+    static extractNodes(contentBlock) {
+        let existingNodes = [];
+
+        for (let key in contentBlock) {
```

Comment:
why do we need to loop over the array twice? Can we not just do this all at once?

Reply:
Ah, good catch! Fixed. 👍